packages/account/src/Sections/Profile/PersonalDetails/personal-details.tsx

```diff
@@ -0,0 +1,680 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Formik, Form, FormikHelpers } from 'formik';
+import { BrowserHistory } from 'history';
+import { withRouter } from 'react-router';
+import {
+    Button,
+    Checkbox,
+    DesktopWrapper,
+    Dropdown,
+    FormSubmitErrorMessage,
+    HintBox,
+    Input,
+    Loading,
+    MobileWrapper,
+    SelectNative,
+    Text,
+} from '@deriv/components';
+import { useGetAccountStatus, useResidenceList, useSettings } from '@deriv/api';
+import { GetSettings } from '@deriv/api-types';
+import { getBrandWebsiteName, routes } from '@deriv/shared';
+import { Localize, localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import LeaveConfirm from 'Components/leave-confirm';
+import FormFooter from 'Components/form-footer';
+import FormBody from 'Components/form-body';
+import FormBodySection from 'Components/form-body-section';
+import { DateOfBirthField } from 'Components/forms/form-fields';
+import FormSubHeader from 'Components/form-sub-header';
+import LoadErrorMessage from 'Components/load-error-message';
+import POAAddressMismatchHintBox from 'Components/poa-address-mismatch-hint-box';
+import { getEmploymentStatusList } from 'Sections/Assessment/FinancialAssessment/financial-information-list';
+import { isServerError } from 'Helpers/utils';
+import { salutation_list } from './constants';
+import InputGroup from './input-group';
+import { getPersonalDetailsInitialValues, getPersonalDetailsValidationSchema, makeSettingsRequest } from './validation';
+import FormSelectField from 'Components/forms/form-select-field';
+
+type TRestState = {
+    show_form: boolean;
+    errors?: boolean;
+    api_error?: string;
+    changeable_fields?: string[];
+    form_initial_values?: Record<string, any>;
+};
+
+export const PersonalDetailsForm = observer(({ history }: { history: BrowserHistory }) => {
+    const [is_submit_success, setIsSubmitSuccess] = React.useState(false);
```

Comment:
Why we need this ? We can get isSuccess from useSettings

Reply:
We need this state because `isSuccess` is showing a green tick button which then will maintain the whole time. While this state will have a timeout where after user submitted their new documents, the button will turn to green tick for 3 seconds then will be coming back to the default `submit` button afterwards.